pyrasterframes/src/main/python/docs/nodata-handling.pymd

```diff
@@ -104,30 +107,30 @@ one_mask = mask_part.withColumn('mask', rf_local_add('nodata', 'defect')) \
 one_mask.select(rf_cell_type('mask')).distinct().show()
 ```
 
-Now we will use the @ref:[`rf_mask_by_value`](reference.md#rf-mask-by-value) to designate the cloudy and other unwanted pixels as NoData in the blue column. Because there is not a NoData already defined, we will choose one. Note that in this particular example the minimum value is greater than zero, so we can use 0 as the NoData value.
+Because there is not a NoData already defined, we will choose one. In this particular example, the minimum value is greater than zero, so we can use 0 as the NoData value.
 
 ```python pick_nd
 one_mask.agg(rf_agg_stats('blue').min.alias('blue_min')).show()
 ```
 
-We can now construct the cell type string for our blue band's cell type, but designating 0 as NoData.
+We can now construct the cell type string for our blue band's cell type, designating 0 as NoData.
 
 ```python get_ct_string
 blue_ct = one_mask.select(rf_cell_type('blue')).distinct().first()[0][0]
 masked_blue_ct = CellType(blue_ct).with_no_data_value(0)
 masked_blue_ct.cell_type_name
 ```
 
-Convert the cell type and apply the mask. Since the mask column is bit type, the addition done above was equivalent to a logical or. So the true values are 1.
+Now we will use the @ref:[`rf_mask_by_value`](reference.md#rf-mask-by-value) to designate the cloudy and other unwanted pixels as NoData in the blue column by converting the cell type and applying the mask.
 
-```python mask_blu 
+```python mask_blu
```

Comment:
i think nowadays (since last friday) the convention is to pass the python `CellType` object into functions like `rf_convert_cell_type`, as opposed to the str typed `cell_type_name` property.

Reply:
I'm getting an error with this

Reply:
Your suggestion worked!